contrib/globalsign/mgo/mgo.go

```diff
@@ -0,0 +1,395 @@
+package mgo
+
+import (
+	"context"
+
+	"github.com/globalsign/mgo"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+type mongoConfig struct {
```

Comment:
Can we move the options and config to an `option.go` to match other integrations?